src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -40,7 +40,7 @@ m4_changequote(`<!', `!>')
 
 def path(schema_madlib, source_table, output_table, partition_expr,
          order_expr, pattern_expr, symbol_expr, agg_func,
-         persist_rows=None, **kwargs):
+         persist_rows=None, overlapping_patterns=None, **kwargs):
```

Comment:
It might be a good idea to set the default value of overlapping_patterns to False instead of None. 
From what I gather, it is better to do that in the path.sql_in file where you can pass False instead of 
NULL in the overloaded function for the overlapping_patterns param. 

Of course, we will still need to have the "if overlapping_patterns is None:" check below to handle 
the case where overlapping_patterns is explicitly set to NULL by the user.